androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/thankyou/ThankYouScreen.kt

```diff
@@ -15,36 +16,54 @@ import androidx.compose.ui.unit.dp
 import co.nimblehq.avishek.phong.kmmic.android.R
 import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
 import com.airbnb.lottie.compose.*
+import kotlinx.coroutines.delay
+
+const val FadeAnimationDurationInMillis = 500
 
 @Composable
 fun ThankYouScreen(
     onComplete: () -> Unit,
 ) {
     val composition by rememberLottieComposition(LottieCompositionSpec.RawRes(R.raw.thank_you))
     val progress by animateLottieCompositionAsState(composition)
+    var shouldShowContent by remember { mutableStateOf(true) }
 
-    Column(
-        verticalArrangement = Arrangement.Center,
+    AnimatedVisibility(
+        visible = shouldShowContent,
+        enter = fadeIn(animationSpec = tween(FadeAnimationDurationInMillis)),
+        exit = fadeOut(animationSpec = tween(FadeAnimationDurationInMillis)),
         modifier = Modifier
             .fillMaxSize()
-            .padding(horizontal = 20.dp)
+            .statusBarsPadding()
+            .navigationBarsPadding()
     ) {
-        LottieAnimation(
-            composition = composition,
+        Column(
+            verticalArrangement = Arrangement.Center,
```

Comment:
The right way to center contents in a `Column` or `Row` is using `verticalArrangement` and `horizontalAlignment`.
```suggestion
            verticalArrangement = Arrangement.Center,
            horizontalAlignment = Alignment.CenterHorizontally,
```